web/src/compose_actions.ts

```diff
@@ -91,6 +91,10 @@ function hide_box(): void {
     compose_fade.clear_compose();
     $(".message_comp").hide();
     $("#compose_controls").show();
+    // Assume a muted recipient row for the next time
+    // the compose box is reopened
+    $("#compose-recipient").addClass("muted-recipient-row");
```

Comment:
I think "muted" is the wrong term to use here; just because a "muted recipient" is a thing. 

Maybe "recipient-row-attention-low" would work?

Reply:
I guess then `maybe_mute_recipient_row` would be `update_recipient_row_attention`?

I think this can be a follow-up rename from my standpoint.